.github/workflows/python-publish-nightly.yml

```
@@ -38,7 +38,12 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Change Version Info
       run: | 
-        python release.py
+        python scripts/release.py
```

```suggestion
        python release.py
```
Moving `release.py` into the `scripts/` directory will limit its access to root files meaning that `setup.py` version info won't be able to be changed.

---

It doesn't make a difference in the script. I tested it on the computer myself.